apps/react/src/features/studios/utils/sortValueFromQueryParams.ts

```diff
@@ -0,0 +1,24 @@
+import { StudioSortDirection } from '@js-camp/core/models/enums/studio-sort-direction.enum';
+import { StudioSort } from '@js-camp/core/models/studio-sort.model';
+
+/**
+ * Transforms sort param from url string format to studio sort model.
+ * @param param Sort param in url string.
+ */
+export function sortValueFromQueryParams(param: string): StudioSort {
+	if (param.length === 0) {
+		return {};
+	}
```

Comment:
Returning an empty object is not the best idea, there is a risk of running into a bug

```suggestion
export function sortValueFromQueryParams(param: string): StudioSort | null {
	if (param.length === 0) {
		return null;
	}
```